saber-config.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
                 { label: "GeoIP", link: "/wiki/GeoIP.html" },
                 { label: "Protect", href: "https://wiki.mc-ess.net/wiki/Configuration_file#EssentialsProtect" },
                 { label: "Spawn", link: "/wiki/Module-Breakdown.html#essentialsx-spawn" },
+                { label: "Discord", link: "/wiki/Module-Breakdown.html#essentialsx-discord" },
```

Comment:
This list is alphabetical:
```suggestion
                { label: "Discord", link: "/wiki/Module-Breakdown.html#essentialsx-discord" },
                { label: "GeoIP", link: "/wiki/GeoIP.html" },
                { label: "Protect", href: "https://wiki.mc-ess.net/wiki/Configuration_file#EssentialsProtect" },
                { label: "Spawn", link: "/wiki/Module-Breakdown.html#essentialsx-spawn" },
```